launcher/src/backend/SSHService.js

```diff
@@ -166,8 +200,8 @@ export class SSHService {
     }
   }
 
-  async exec(command, useSudo = true) {
-    const ensureSudoCommand = "sudo -u 'root' -i <<'=====EOF'\n" + command + "\n=====EOF";
+  async exec(command, useSudo = true, useRoot = true) {
```

Comment:
as far as i see it you're not using this parameter, so i think we can delete it.